src/cond.test.ts

```diff
@@ -0,0 +1,32 @@
+import { cond } from './cond';
+import { pipe } from './pipe';
+
+describe('data first', () => {
+  test('should return value of first pair', () => {
+    const value = cond('Jokic', [
+      [s => s === 'Murray', s => 'point guard'],
```

Comment:
Use an empty params list to avoid this lint
```suggestion
      [s => s === 'Murray', () => 'point guard'],
```